src/Agent.Sdk/Util/VssUtil.cs

```diff
@@ -21,6 +21,30 @@ public static class VssUtil
     {
         private static UtilKnobValueContext _knobContext = UtilKnobValueContext.Instance();
 
+        public static bool IsCustomServerCertificateValidationSupported
+        {
+            get
+            {
+                using (var handler = new HttpClientHandler())
+                {
+                    handler.ServerCertificateCustomValidationCallback = (message, cert, chain, errors) => { return true; };
+
+                    using (var client = new HttpClient(handler))
+                    {
+                        try
+                        {
+                            client.GetAsync("https://microsoft.com/").GetAwaiter();
```

Comment:
```suggestion
                            client.GetAsync("https://microsoft.com/").GetAwaiter().GetResult();
```

Reply:
Could you please also extract "https://microsoft.com/" into a separate constant inside the class?